docs/MO_DG/prepare_model/convert_model/Cutting_Model.md

```diff
@@ -1,15 +1,15 @@
 # Cutting Off Parts of a Model  {#openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model}
 
-Sometimes some parts of a model must be removed while the Model Optimizer is converting models to the Intermediate Representation. This chapter describes methods of doing cutting off parts of a model using Model Optimizer command-line options. Model cutting applies mostly to TensorFlow\* models, but is also useful for other frameworks. In this chapter, TensorFlow examples are used for illustration.
+Sometimes, some parts of a model must be removed while the Model Optimizer converts models to the Intermediate Representation. This chapter describes methods of cutting off parts of a model using Model Optimizer command-line options. Model cutting applies mostly to TensorFlow models, but it is also useful for other frameworks. TensorFlow examples are used in this chapter for illustrative purposes.
 
 ## Purpose of Model Cutting
 
 The following examples are the situations when model cutting is useful or even required:
 
 *   model has pre- or post-processing parts that cannot be translated to existing OpenVINO operations.
-*   model has a training part that is convenient to be kept in the model, but not used during inference.
-*   model is too complex (contains lots of unsupported operations that cannot be easily implemented as custom layers), so the complete model cannot be converted in one shot.
-*   problem with model conversion in the Model Optimizer or inference in the OpenVINO Runtime occurred. To localize the issue, limit the scope for conversion by iteratively searching for problematic places in the model.
+*   model has a training part that is convenient to be kept in the model but not used during inference.
+*   model is too complex (contains a lot of unsupported operations that cannot be easily implemented as custom layers), so the complete model cannot be converted in one go.
+*   problem with model conversion in Model Optimizer or inference in the OpenVINO™ Runtime occurred. To identify the issue, limit the scope for conversion by iterative search for problematic areas in the model.
 *   single custom layer or a combination of custom layers is isolated for debugging purposes.
```

Comment:
```suggestion
*   A model has a training part that is convenient to be kept in the model but not used during inference.
*   A model is too complex be converted at once, because it contains a lot of unsupported operations that cannot be easily implemented as custom layers.
*   A problem occurs with model conversion in Model Optimizer or inference in OpenVINO™ Runtime. To identify the issue, limit the conversion scope by iterative search for problematic areas in the model.
*   A single custom layer or a combination of custom layers is isolated for debugging purposes.
```